OracleUnifiedDirectorySM/LICENSE.txt

```diff
@@ -0,0 +1,35 @@
+Copyright (c) 2019-2020, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
This file should have been ignored via `.gitignore`. I recommend removing it completely from your fork.